packages/jest-preset-mc-app/stylelint-runner/run.js

```diff
@@ -0,0 +1,78 @@
+const { pass, fail } = require('create-jest-runner');
+const stylelint = require('stylelint');
+const postcssImport = require('postcss-import');
+const postcssCustomMediaQueries = require('postcss-custom-media');
+const cssVariables = require('postcss-css-variables');
```

Comment:
for some reason, postcss-custom-properties messes with the spacing. Maybe it makes sense just to use this plugin everywhere instead.